common/setup.py

```diff
@@ -25,18 +25,26 @@ def get_changelog():
     setup(
       name="devpi-common",
       description="utilities jointly used by devpi-server and devpi-client",
-      long_description="\n\n".join([README, CHANGELOG]),
```

Comment:
Why did you remove the changelog? We have it here explicitly, so people can see the last changes easily from the pypi page.

Reply:
After installing the package using `yum`, I used to see the rpm spec. Imo having change log in the rpm spec make no sense. `rpm -qi devpi-common`.